.dotnet/src/Custom/FineTuning/FineTuningClient.cs

```diff
@@ -63,5 +63,66 @@ protected internal FineTuningClient(ClientPipeline pipeline, Uri endpoint, OpenA
     {
         _pipeline = pipeline;
         _endpoint = endpoint;
+    }
+
+    internal virtual CreateJobOperation CreateCreateJobOperation(string jobId, string status, PipelineResponse response)
+    {
+        return new CreateJobOperation(_pipeline, _endpoint, jobId, status, response);
+    }
+
+    internal virtual PipelineMessage CreateCreateFineTuningJobRequest(BinaryContent content, RequestOptions options)

```

Comment:
Same comment as https://github.com/annelo-msft/openai-in-typespec/pull/11/files#r1752485342

Reply:
moved